easybuild/easyconfigs/a/ABINIT/ABINIT-8.4.4-intel-2017b.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = "8.4.4"
+
+homepage = 'http://www.abinit.org/'
+description = """ABINIT is a package whose main program allows one to find the total energy,
+ charge density and  electronic structure of systems made of electrons and nuclei (molecules
+ and periodic solids) within Density Functional  Theory (DFT), using pseudopotentials and a
+ planewave or wavelet basis."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'usempi': True}
```

Comment:
Please also enable the use of `-fPIC` (to dance around the problem with the Intel compilers & recent `glibc` versions).

```python
toolchainoptions = {'usempi': True, 'pic': True}
```

I double-checked, this also fixes the problem with the tests, just like defining `$LD_BIND_NOW` does...